decentraland/unity-renderer
# feat: backpack throttling

## What does this PR change?

- Limited the amount of Thumbnail that load concurrently to 10.
- Limited the number of Wearables that can be shown to 3 rows and X columns ( where X is based on the screen size )
- If the Wearables are higher than the limit that can be shown, pagination is shown.

- Merged NFTItemToggle prefab variants into one, the variance is now driven by skins defined by `NFTItemToggleSkin` ScriptableObjects
- Removed the CollectionContainer prefab as it has no purpose now, the base script still remains and its called `ItemToggleContainer` 

- The ThumbnailManager manages the queue for the loading of the Textures, those textures are being cached and never get destroyed, we might have to limit or change this in the future.

## How to test the changes?

- Open the backpack, and check that changing the resolution also changes the number of wearables that can be shown. (the shirts section is the one with the most wearables)
- Check that the pagination appears and disappears when changing wearable sections.
- Check that the pagination works correctly, and has sound feedback and subtle animations.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
